buildkitd/settings.go

```diff
@@ -28,6 +28,7 @@ type Settings struct {
 	ServerTLSKey         string
 	UseTCP               bool
 	UseTLS               bool
+	VolumeName           string
```

Comment:
Should `ContainerName` be part of settings too? 🤔

Reply:
I considered that; however we need the container name when making a call such as `GetSettingsHash(ctx, containerName)`.